plugins/pulumi/config.ts

```diff
@@ -17,7 +17,7 @@ import { defaultPulumiVersion, supportedVersions } from "./cli"
 type PulumiProviderConfig = GenericProviderConfig & {
   version: string | null
   previewDir: string | null
-  orgName?: string
+  orgName: string
```

Comment:
Shouldn't `orgName` stay optional too?

Reply:
If orgName is null or a string (like defined in joi) it should be
```suggestion
  orgName: string | null
```

but I think it would be better to avoid null if possible, and keep using undefined in case it isn't defined (keep `orgName?: string`